app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { Table as ContactList } from "neetoui";
+
+import { ROW_DATA, contactListData } from "./utils";
+
+const Table = () => (
```

Comment:
This is not the right way to import under an alias when there's a conflict in component names. Refer: https://academy.bigbinary.com/learn-react/react-best-practices/miscellaneous-tips#:~:text=Importing%20under%20an%20alias

Reply:
Will do the change.
As per React Best Practices, the ``` Table ``` should be aliased as ``` NeetoUITable ```.